dist/src/main/resources/wildfly/patches/standalone.xsl

```diff
@@ -349,7 +354,7 @@
                  numAvailSchedulerThreads="3">
         <xsl:attribute name="enabled">
           <xsl:choose>
-            <xsl:when test="$kettle.build.type='dev'">true</xsl:when>
+            <xsl:when test="$kettle.build.type='dev'">${hawkular.agent.enabled:true}</xsl:when>
```

Comment:
no need, http://www.w3schools.com/xsl/el_text.asp

yes means no escaping of special chars `<` -> `&lt;`.
Here the values that makes sense for this parameter is `true` and `false` and if some possible attacker wants to inject some tricky xml snippet by some kind of xml injection it's imho better to escape the 'picitka' (<> :])